client/src/components/_custom/drp/utils/utils.js

```diff
@@ -5,6 +5,20 @@ export const formatLabel = (key) => {
     if (typeof key !== 'string') {
         return key;
     }
+
+    // Check if it's a valid URL
```

Comment:
```suggestion
```

This comment seems superfluous given the self-explanatory function name `isValidUrl`.